shared/video-sdk/develop/spatial-audio/project-implementation/web.mdx

```diff
@@ -0,0 +1,106 @@
+<PlatformWrapper platform="web">
+
+### Spatial audio for remote users
+
+When subscribing to a remote user, realize the user's spatial sound effect as follows:
+
+1. Call [`createProcessor`](#createprocessor) to create a `SpatialAudioProcessor` instance.
```

Comment:
So? 

Where is the code example, this does not fit the pattern. 

. Some kind of explanation
   code example. 


I'm stopping here. Can you have a quick look at the Android guide and update to look like that. Soon as that is done I will review again.